ui/selectors/selectors.js

```diff
@@ -647,6 +649,11 @@ export function getShowTestNetworks(state) {
   return Boolean(showTestNetworks);
 }
 
+/**
+ * @deprecated - use getTestNetworks to retrieve image URLs for test
+ * network icons.
```

Comment:
Any additions or changes needed in this message, @georgewrmarshall?

Reply:
I think this is fine!